From 1d00ce523590aa92b43580cc1c23ed0e93e7f75d Mon Sep 17 00:00:00 2001
From: Yeongcheol Kim <aree91@naver.com>
Date: Tue, 8 Mar 2022 11:44:18 +0900
Subject: [PATCH] Auto repair

---
 example.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/example.c b/example.c
index 927aa6c..0f00709 100644
--- a/example.c
+++ b/example.c
@@ -2,7 +2,7 @@ int func1(int x) {
     int y = 100;
     y = x;
     if (x >= 100) { // correct code: if (x > 100) {
-        return -y;
+        return -y + 3;
     }
     return y;
 }
\ No newline at end of file
-- 
2.35.1.windows.2

