From 5e947048ce7f1eb259ee9d669a870ebccf65af42 Mon Sep 17 00:00:00 2001
From: Yeongcheol Kim <aree91@naver.com>
Date: Tue, 8 Mar 2022 11:45:16 +0900
Subject: [PATCH] Auto repair

---
 example.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/example.c b/example.c
index 927aa6c..ded3164 100644
--- a/example.c
+++ b/example.c
@@ -1,6 +1,6 @@
 int func1(int x) {
     int y = 100;
-    y = x;
+    y = -x;
     if (x >= 100) { // correct code: if (x > 100) {
         return -y;
     }
-- 
2.35.1.windows.2

